generate/state_test.go

```diff
@@ -225,6 +225,113 @@ func TestFromState_AWS(t *testing.T) {
 		require.NotNil(t, cfg)
 		assert.Len(t, g.Nodes, 2)
 	})
+	t.Run("Version3", func(t *testing.T) {
+		src, err := ioutil.ReadFile("./testdata/version_3_state.json")
```

Comment:
Is it worth checking a file without modules too?

Reply:
I would say it's not needed, if there are 0 modules it'll have a `""` module (found it while fixing a bug on IM) so there are "always" modules.

But if you have an example I can add it to the test, more tests better haha.